akka-stream/src/main/scala/akka/stream/ssl/StreamTls.scala

```diff
@@ -0,0 +1,85 @@
+package akka.stream.ssl
+
+import akka.stream.impl.StreamLayout.Module
+import akka.stream._
+import akka.stream.ssl._
+import akka.util.ByteString
+import javax.net.ssl.SSLContext
+
+object StreamTls {
```

Comment:
Hm, looking more and more at these I found it less and less nice to have StreamXXX.